src/quicktype-core/language/Swift.ts

```diff
@@ -490,7 +490,7 @@ export class SwiftRenderer extends ConvenienceRenderer {
         this.emitLine("typealias ", name, " = ", this.swiftType(t, true));
     }
 
-    private getProtocolString(): Sourcelike {
+    protected getProtocolsArray(_t: Type): String[] {
```

Comment:
The return type needs to be lowercase `string[]`.

Reply:
Done! Didn't knew about that distinction 😅